src/plugins/intel_cpu/src/nodes/executors/dnnl/dnnl_transpose.hpp

```diff
@@ -0,0 +1,40 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include "nodes/executors/transpose.hpp"
+
+namespace ov {
+namespace intel_cpu {
+
+class DNNLTransposeExecutor : public TransposeExecutor {
+public:
+    explicit DNNLTransposeExecutor(const ExecutorContext::CPtr context);
```

Comment:
```suggestion
    using TransposeExecutor::TransposeExecutor;
```
Constructor inheritance?

Reply:
fixed